transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py

```diff
@@ -156,13 +157,17 @@ def df_to_services(revision: DatasetRevision, df: pd.DataFrame) -> Iterator[Serv
         if pd.isnull(end_date):
             end_date = None
         line_names = record["line_names"]
+        service_type = record["service_type"]
+        if pd.isnull(service_type):
```

Comment:
Should we be storing the record for which the service type was not identified? Seems okay to store it as None but if the validation has skipped this could we raise an exception so that an error is displayed?

Reply:
This would be part of the schema check itself. So, this None check or even an exception is redundant.